portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,60 @@
 <html>
   <head>
     <meta charset="UTF-8">
+    <img src="/Images/Name.png" alt="Sami Alves">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+    <div class="content">
+      <h1>
+        About Me: 
+        <a href="https://www.github.com/Sami-2000">Github</a>
+        <a href="https://www.linkedin.com/in/samantha-alves-865093195/">LinkedIn</a>
+      </h1>
+      <img src="/Images/Trapeze.jpg" alt="Me doing trapeze.">
+      <p>Hi all! My name is Sami and I am a Junior at Yale University studying Physics and Computer Science. 
+          I love both exploring and codifying the patterns that govern our world. When I am not studying, I 
+          work in an interdisciplinary Neuroscience lab studying fly behavior, give tours to
+          perspective students, and tutor students in an introductory Physics class.</p>
+      <p>I also love practicing on the Club Gymnastics team, planning events for the Women in Physics group, volunteering 
+          in local elementary schools, and debating in the Yale Political Union. </p>
+      <p>To see what else I have been up to during quarantine, <a href="photos.html">Click here!</a></p>
+      <p>To read about my hidden talents, <a href="blog.html">Click here!</a></p>
+      <a id="secret" href="bubble.html">Wanna relax?</a>
```

Comment:
```suggestion
      <a id="secret" href="bubble.html">Want to relax?</a>
```

Reply:
Avoid slang or improper English.

Reply:
Thank you for your comments about typos and text conventions as well! In the future, I will try to write text out somewhere with auto-correct (like a google doc) before copying into my html file.